src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -58,10 +63,43 @@ public void ConfigureServices(IServiceCollection services)
     services.AddOpenTelemetry()
         .WithTracing(builder => builder
             .AddAspNetCoreInstrumentation()
-            .AddJaegerExporter());
+            .AddOtlpExporter());
```

Comment:
Any specific reason to use otlp here? If not, can we use console exporter? (Http one uses console)

Reply:
missed the note on PR description. Could you please explain why?
`Console exporter from Http example is not good idea for the ASP.NET Core application`

Reply:
Welll, my mind is leaving in the world where ASP.NET means IIS...

Fixed :- )